Application/EdFi.Ods.Api/Middleware/InstanceSpecificRouteContextMiddleware.cs

```diff
@@ -10,12 +10,12 @@
 
 namespace EdFi.Ods.Api.Middleware
 {
-    public class InstanceYearSpecificMiddleware : IMiddleware
+    public class InstanceSpecificRouteContextMiddleware : IMiddleware
     {
         private const string Pattern = @"^[A-Za-z0-9-]+$";
         private readonly IInstanceIdContextProvider _instanceIdContextProvider;
 
-        public InstanceYearSpecificMiddleware(IInstanceIdContextProvider instanceIdContextProvider)
+        public InstanceSpecificRouteContextMiddleware(IInstanceIdContextProvider instanceIdContextProvider)
```

Comment:
This should really be InstanceIdRouteContextMiddleware to match the naming of the SchoolYear one.

Reply:
done